core/vm/interpreter.go

```diff
@@ -59,6 +59,9 @@ func NewEVMInterpreter(evm *EVM) *EVMInterpreter {
 	case evm.chainRules.IsPrague:
 		// TODO replace with prooper instruction set when fork is specified
 		table = &pragueInstructionSet
+		if err := EnableEIP(2935, table); err != nil {
+			log.Error("EIP 2935 activation failed", "error", err)
```

Comment:
Not sure about `log.Error` here but a panic -- def if this happens won't realize by looking at logs. I'm not a fan of panics, so up to you.

Reply:
```suggestion
			log.Error("EIP 2935 activation failed", "error", err)
			panic(err)
```